js/signin.js

```diff
@@ -47,18 +47,32 @@ import {
   signForm.addEventListener("submit", submitForm);
   function submitForm(event) {
     event.preventDefault();
-  
-    const isTestUser =
-      emailInput.value === TEST_USER.email && passwordInput.value === TEST_USER.password;
-  
-    if (isTestUser) {
-      location.href = "/folder";
-      return;
-    }
-    setInputError({ input: emailInput, errorMessage: emailErrorMessage }, "이메일을 확인해주세요.");
-    setInputError(
-      { input: passwordInput, errorMessage: passwordErrorMessage },
-      "비밀번호를 확인해주세요."
-    );
+
+    fetch("https://bootcamp-api.codeit.kr/api/sign-in", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json" ,
+      },
+      body: JSON.stringify({
+        email: emailInput.value,
+        password: passwordInput.value,
+      }),
+    })
+    .then(async (response) => {
+      if (response.status === 200) {
```

Comment:
`response.status` 부분은 로직이 복잡해지면서 다양한 분기처리에 활용될 수 있습니다. 확장성을 위해 별도 변수로 분리하는 것도 좋아보이네요.
```suggestion
const statusCdoe = response.status
```